ui/components/ui/form-combo-field/form-combo-field.tsx

```diff
@@ -14,7 +14,7 @@ import { I18nContext } from '../../../contexts/i18n';
 import { Display, IconColor } from '../../../helpers/constants/design-system';
 
 export interface FormComboFieldOption {
-  value?: string;
+  value: string;
```

Comment:
I agree `value` makes sense as the required property, but then should we make `primaryLabel` optional so an alternate label isn't forced but supported if needed?

At which point we'd default the `primaryLabel` to match the value if not specified.

Reply:
Absolutely, Matt! Making display text optional with a default to the value streamlines our code and keeps things flexible. Done.